### PR TITLE
OCPVE-346: chore: move all controllers under `internal/controllers`

### PR DESCRIPTION
**Pull request by jakobmoellerdev**
Moves all controllers (including vgmanager) under `internal/controllers` after templating recommendations based on https://github.com/kubernetes-sigs/kubebuilder/tree/master/testdata/project-v4

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ If you encounter a failure message such as `no available devices found` while in
 $ lsblk --paths --json -o NAME,ROTA,TYPE,SIZE,MODEL,VENDOR,RO,STATE,KNAME,SERIAL,PARTLABEL,FSTYPE
 ```
 
-This prints information about the disks on the host. Review this information to see why a device is not considered available for LVMS utilization. For example, if a device has partlabel `bios` or `reserved`, or if they are suspended or read-only, or if they have children disks or `fstype` set, LVMS considers them unavailable. Check [filter.go](../pkg/vgmanager/filter.go) for the complete list of filters LVMS makes use of.
+This prints information about the disks on the host. Review this information to see why a device is not considered available for LVMS utilization. For example, if a device has partlabel `bios` or `reserved`, or if they are suspended or read-only, or if they have children disks or `fstype` set, LVMS considers them unavailable. Check [filter.go](../pkg/controllers/vgmanager/filter.go) for the complete list of filters LVMS makes use of.
```
Comment (suleymanakbas91) [inline docs/troubleshooting.md:66]: Should be inside `internal`. The link does not work now.

File: internal/cluster/namespace.go [hunk 2]
```diff
@@ -7,9 +7,9 @@ import (
 
 const OperatorNamespaceEnvVar = "POD_NAMESPACE"
 
-// GetOperatorNamespace returns the Namespace the operator should be watching for changes
+// GetOperatorNamespace returns the GetNamespace the operator should be watching for changes
```
Comment (suleymanakbas91) [inline internal/cluster/namespace.go:10]: I guess these Namespace -> GetNamespace changes were done unintentionally.

File: internal/controllers/constants/constants.go [hunk 3]
```diff
@@ -25,7 +25,7 @@ const (
 
 	// CSI Controller container and deployment names
 	TopolvmControllerDeploymentName = "topolvm-controller"
-	TopolvmControllerContainerName  = "topolvm-controller"
+	TopolvmControllerContainerName  = TopolvmControllerDeploymentName
```
Comment (suleymanakbas91) [inline internal/controllers/constants/constants.go:28]: We can just delete `TopolvmControllerContainerName` and use `TopolvmControllerDeploymentName` instead.

File: internal/controllers/lvmcluster/lvmcluster_controller.go [hunk 4]
```diff
@@ -25,13 +25,14 @@ import (
 
 	configv1 "github.com/openshift/api/config/v1"
 	lvmv1alpha1 "github.com/openshift/lvm-operator/api/v1alpha1"
-	"github.com/openshift/lvm-operator/pkg/cluster"
+	"github.com/openshift/lvm-operator/internal/cluster"
+	"github.com/openshift/lvm-operator/internal/controllers/constants"
+	resource2 "github.com/openshift/lvm-operator/internal/controllers/lvmcluster/resource"
```
Comment (suleymanakbas91) [inline internal/controllers/lvmcluster/lvmcluster_controller.go:30]: why is there a 2 in the end? :)

**Review by suleymanakbas91 — CHANGES_REQUESTED** [review]

File: internal/cluster/namespace.go (hunk 2 above)
Comment (jakobmoellerdev) [reply internal/cluster/namespace.go]: yup, good catch... generic refactoring mistake

File: internal/controllers/lvmcluster/lvmcluster_controller.go (hunk 4 above)
Comment (jakobmoellerdev) [reply internal/controllers/lvmcluster/lvmcluster_controller.go]: i wonder that myself :D

**Pull request merged**